### DISCUSSION TITLE
Search?

### DISCUSSION THREAD

**Discussion by jtbayly** [markdoc/markdoc#4] [Q&A] [answered]
I just looked, and I couldn’t find any way to search the docs site. This seems like a pretty important thing to me. Did I just miss it?

**Comment by mfix-stripe** [comment, marked as answer]
Coming very shortly! Thanks @jtbayly.

**Reply by mfix-stripe** [reply]
Search is live!

**Reply by jtbayly** [reply]
Is there any documentation for it yet? How would I add search to a markdoc site from scratch?

I see that it's live on MarkDoc.io with Algolia. That's a bit sad to me, as I was hoping for a static solution such as [Lunr](https://lunrjs.com) or [Fuse](https://fusejs.io). And Algolia adds a cost, not just another server.

**Comment by weisisheng** [comment]
Speaking from the "cheap seats", would love to see a few options here for those who have them.

**Reply by joshuataylor** [reply]
Not just cheapseats, it's also for internal documentation etc.

**Reply by kryltsov** [reply]
Am I right saying that https://markprompt.com/ will keep your org docs in own vector database and feed to OpenAI as prompts? As a result, you will have to trust Algolia to keep your internal docs secure. Does Stripe use Algolia to provide internal docs search if I may ask?

**Reply by mfix-stripe** [reply]
I'm not exactly sure how  https://markprompt.com/ works, but it's open source, so you could ask our @motifland friends over in their repo: https://github.com/motifland/markprompt

As for our internal docs, no, we don't use Algolia—we use an internal search API.
